### PR TITLE
Updated CHANGELOG and pacakge.json for 3.12.0 release

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.12.0 (2020-04-01)
+
+- Updated the SKY UX icon font to 3.0.0-beta.1. [#105](https://github.com/blackbaud/skyux-theme/pull/105)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]: Is `font` the right word here? It seems like it should be `version`? I might just not be following, but updating the `font to` something makes me expect to see the font name that we are updating to.

Comment (Blackbaud-PaulCrowder) [reply CHANGELOG.md]: I changed it to "SKY UX icon font version" to be more explicit.

Comment (Blackbaud-PaulCrowder) [reply CHANGELOG.md]: By the way, the name of the font is "SKY UX icons" or `skyux-icons`.

Comment (blackbaud-johnly) [reply CHANGELOG.md]: Thanks for the clarification. I wasn't sure if font was the right word at all. Clearly it is. Thanks!

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-3.12.0](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-3.12.0)

**Pull request merged**